package.json

```
@@ -41,6 +41,7 @@
     "mdast-util-definitions": "^5.0.0",
     "mdurl": "^1.0.0",
     "micromark-util-sanitize-uri": "^1.0.0",
+    "trim-lines": "^3.0.1",
```

```suggestion
    "trim-lines": "^3.0.0",
```

Using loose semver ranges makes it easier for package managers to install things and keeps lock file diffs smaller.

---

Semver solves this.